dkandalintsev/login.feature

```diff
@@ -0,0 +1,136 @@
+
+Background:
+    Given user does not have saved Google accounts in browser
+    And user clicks "Sign in with Google" button
+
+Scenario: 1.09 User tries to sign in with non-existing email
+    Given user sees "Email or phone" input
+    When user fills in "Email or phone" input with <non-existing email>
+    Examples:
+        | non-existing email |
+        | ifmnvme@gmail.com  |
+    And user clicks "Next"
+    Then email input is outlined with red color
+    And user sees "Couldn't find your Google account" message under the input
+
+Scenario: 1.10 User tries to sign in with non-existing phone number
+    Given user sees "Email or phone" input
+    When user fills in "Email or phone" input with <non-existing phone>
+    Examples:
+        | non-existing phone |
+        | +79819139699       |
+    And user clicks "Next"
+    Then email input is outlined with red color
+    And user sees "Couldn't find your Google account. Try using your email address instead" message under the input
+
+Scenario: 1.11 User tries to sign in with incorrect password
+    Given user fills in "Email or phone" input with <email address>
+    And user clicks "Next"
+    When user fills in "Enter your password" input with <incorrect password>
+    Examples:
+        | incorrect password |
+        | Trewqasdf4321?     |
+    And user clicks "Next"
+    Then password input is outlined with red color
+    And user sees "Wrong password. Try again or click ‘Forgot password’ to reset it." message under the input
+
+Scenario: 1.12 User signs in with correct email and password
+    Given user fills in "Email or phone" input with <email address>
+    And user clicks "Next"
+    And user fills in "Enter your password" input with <password>
+    When user clicks "Next"
+    Then user is on the Quantori Scoring main page
+
+Scenario: 1.13 User does not fill email input
+    Given user leaves "Email or phone" input empty
+    When user clicks "Next"
+    Then email input is outlined with red color
+    And user sees "Enter an email or phone number" message under the input
+
+Scenario: 1.14 User does not fill password input
+    Given user fills in "Email or phone" input with <email address>
+    And user clicks "Next"
+    And user leaves "Enter your password" input empty
+    When user clicks "Next"
+    Then password input is outlined with red color
+    And user sees "Enter a password" message under the input
+
+Scenario: 1.15 User tries to log in with invalid data
+    Given user sees "Email or phone" input
+    When user fills in "Email or phone" input with <invalid data>
+    Examples:
+        | invalid data |
+        | +73455829    |
+        | kmegjbmp@rfg |
+    And user clicks "Next"
+    Then email input is outlined with red color
+    And user sees "Enter a valid email or phone number" message under the input
+
+Scenario: 1.16 Username is saved
+    Given user is successfully signed in
+    And user changed his username in Quantori Scoring to <username>
```

Comment:
Hm, do you test "Quantori Scoring"'s correct work?:)

Reply:
Yeah, I thought that it is a part of login feature, but on the second thought, I think I'll remove this scenario